QUEST.tsv

```diff
@@ -300,203 +300,203 @@ QUEST_20150317_000299	No. Try again.
 QUEST_20150317_000300	Good. Very good!
 QUEST_20150317_000301	You got that wrong. {nl}Why don't you try again?
 QUEST_20150317_000302	The place upstairs is the divine place where the huge statue of Goddess, the symbol of Fedimian, is located. {nl}All believers, tidy yourself and worship quietly.
-QUEST_20150317_000303	Fedimian became like this four years ago…{nl}It was such a beautiful and vibrant city.{nl}It is fortunate that the divine district, where the central statue of the goddess is located, is still okay, at least.
-QUEST_20150317_000304	Fedimian was originally divided into two districts - the commerce district and the divine district.{nl}But due to the incident that happened four years ago, the commerce district was completely destroyed…{nl}
-QUEST_20150317_000305	Well, at least the divine district is still left. {nl}That's all because of the blessing of the goddess.
-QUEST_20150317_000306	I am just an ordinary old man in Fedimian. {nl}If you want to know more, bring me some potato dumplings…
-xQUEST_20150317_000307	The food at Fedimian is too awful. {nl}Before I die, is it possible to try all delicious food in the world?
-QUEST_20150317_000308	This is the divine district in Fedimian. {nl}May the goddess bless you in the future.
-xQUEST_20150317_000309	Right now, the commerce district is closed for reconstruction.{nl}Please do not enter the district.
+xQUEST_20150317_000303	Fedimian became like this four years ago…{nl}It was such a beautiful and vibrant city. {nl}Fortunately, the divine district, where the central statue of the Goddess is located, is still okay.
+xQUEST_20150317_000304	Fedimian was originally divided into two districts, the commerce district and the divine district. {nl}But an incident left the commerce district completely destroyed…
```

Comment:
Lost the detail that the incident was 4 years ago. This line makes it sound like it was just another incident